tests/smartswitch/platform_tests/test_platform_dpu.py

```diff
@@ -152,10 +154,14 @@ def test_restart_pmon(duthosts, dpuhosts, enum_rand_one_per_hwsku_hostname,
     pmon_status = check_pmon_status(duthost)
     pytest_assert(pmon_status == 1, "PMON status is Not UP")
 
-    post_test_dpus_check(duthost, dpuhosts, dpu_on_list, ip_address_list, num_dpu_modules, "Non-Hardware")
+    post_test_dpus_check(duthost, dpuhosts,
+                         dpu_on_list, ip_address_list,
+                         num_dpu_modules, "Switch rebooted DPU")
 
 
-def test_system_health_state(duthosts, enum_rand_one_per_hwsku_hostname,
+@pytest.mark.disable_loganalyzer
```

Comment:
We shouldn't expect any syslog errors when the DPU is shutdown by the CLI, why disable the loganalyzer?